cli/package.json

```diff
@@ -40,10 +40,29 @@
     "yaml": "^2.3.1"
   },
   "scripts": {
+    "build": "tsc -b",
```

Comment:
Can you use --project to use the build config and verify to spec files are in the output?